interpreter/cling/include/cling/Interpreter/DynamicLibraryManager.h

```diff
@@ -34,6 +34,16 @@ namespace cling {
       kLoadLibNumResults
     };
 
+    /// Describes the library search paths.
+    struct SearchPathInfo {
+      /// The search path.
+      ///
+      std::string Path;
+
+      /// True if the Path is on the LD_LIBRARY_PATH.
+      ///
+      bool IsUser;
```

Comment:
System libs can be found through `LD_LIBRARY_PATH`, too... Maybe rename `bool FromLDLibPath`?

Reply:
The presumption is that the user specified the LD_LIBRARY_PATH which may turn a system library in a user library.